### PR TITLE
docs: update link to CloudFormation Template

### PR DESCRIPTION
**Pull request by bryankaraffa**
### Description

Update toplevel README to update link to CloudFormation Template

### Contribution Check List

- [x] All tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Reference
 
 ##### Credential Setup
 
-- [Read Only CFT](./cost/aws/FlexeraReadOnlyPolicy.template)
+- [Flexera Automation CloudFormation Template](./tools/cloudformation-template)
```
Comment (nia-vf1) [inline README.md:62]: @bryankaraffa - quick question, previously you were linking to the actual template, but now you are not (which makes sense as it now directs to the README). Just wanted to make sure this was intentional

Comment (bryankaraffa) [reply README.md:62]: Yeah I think linking to README will be a better UX for customers.  README has links to the CFT, and also steps for how to use it (including 1-click deploy link)

Comment (bryankaraffa) [reply README.md:62]: bump @nia-vf1 , let me know if there's any other Q's.  Thank you!

**Comment by bryankaraffa** [comment]
bump @taterdog590 for PR.  Thank you!

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Pull request merged**